### ISSUE TITLE
Add MPRIS D-Bus interface integration for adding song information

### ISSUE THREAD

**Issue by nattyebola** [NickvisionApps/Cavalier#143]
it's all in the title, this application is gorgeous, by the way.
https://specifications.freedesktop.org/mpris-spec/latest/

**Comment by fsobolev** [comment]
Sorry, but I don't quite understand your request. Cavalier itself doesn't play any sound, so there's no info to send to MPRIS. If you use Cavalier while listening to music using player that doesn't provide MPRIS integration, it should be addressed to the player you use. Or do you want to display info from MPRIS in Cavalier window? In this case, not sure how it should look, any ideas?

**Comment by nattyebola** [comment]
Yes, it's to be able to display the current playback in cavalier. If it were possible to choose the font, the size and where to display this information, that would be great.
An option to scroll the text like an LCD display would be a plus =)
